assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -87,15 +88,32 @@ function DashboardClicksWidget() {
 	if ( ! data || ! data.length ) {
 		return getNoDataComponent( _x( 'Search Console', 'Service name', 'google-site-kit' ) );
 	}
+	// Split the data in two chunks.
+	const half = Math.floor( data.length / 2 );
+	const latestData = data.slice( half );
+	const olderData = data.slice( 0, half );
 
-	const { totalClicks, totalClicksChange, dataMap } = extractSearchConsoleDashboardData( data );
+	const totalClicks = sumObjectListValue( latestData, 'clicks' );
+	const totalOlderClicks = sumObjectListValue( olderData, 'clicks' );
+	const totalClicksChange = changeToPercent( totalOlderClicks, totalClicks );
 
+	// Sparkline data needs headers and dates formatted as MM/DD
+	const sparklineData = [
+		[
+			{ type: 'string', label: 'Day' },
+			{ type: 'number', label: 'Clicks' },
+		],
+		...extractForSparkline( latestData, 'clicks', 'keys.0' ).map( ( row ) => {
+			const date = new Date( row[ 0 ] );
+			return [ date.getMonth() + 1 + '/' + date.getUTCDate(), row[ 1 ] ];
```

Comment:
Let's move the comment closer to what it's referring to (applies elsewhere too)
```suggestion
			// Sparkline data needs headers and dates formatted as MM/DD
			return [ date.getMonth() + 1 + '/' + date.getUTCDate(), row[ 1 ] ];
```